mobile-widgets/qml/DownloadFromDiveComputer.qml

```diff
@@ -127,18 +137,22 @@ Kirigami.Page {
 					currentIndex = manager.getDetectedProductIndex(comboVendor.currentText)
 				}
 			}
-			Controls.Label { text: qsTr(" Connection:") }
+			Controls.Label {
+				text: qsTr(" Connection:")
+				font.pointSize: subsurfaceTheme.regularPointSize
+			}
 			Controls.ComboBox {
 				id: comboConnection
 				Layout.fillWidth: true
+				Layout.preferredHeight: fontMetrics.height * 2.5
 				model: connectionListModel
 				currentIndex: -1
 				delegate: Controls.ItemDelegate {
 					width: comboConnection.width
+					height: fontMetrics.height * 2.0
 					contentItem: Text {
 						text: modelData
-						// color: "#21be2b"
-						font.pointSize: subsurfaceTheme.smallPointSize
+						font.pointSize: subsurfaceTheme.regularPointSize
```

Comment:
this had been intentional as the connection text is so long in most cases and with the smaller font the chance of people being able to see the relevant part seemed higher.

Reply:
Undid the font change here, and force pushed the PR again.

Reply:
thanks. I'll merge without waiting for Travis... I like to live on the edge...